src/js/common/schemaform/SaveInProgressIntro.jsx

```diff
@@ -49,8 +61,8 @@ export default class SaveInProgressIntro extends React.Component {
         <div>
           <div className="usa-alert usa-alert-info schemaform-sip-alert">
             <div className="usa-alert-body">
-              You can save this form, and come back later to finish filling it out.<br/>
-              <button className="va-button-link" onClick={() => this.props.toggleLoginModal(true)}>Sign in to save your form in progress.</button>
+              You can save this form in progress, and come back later to finish filling it out.<br/>
```

Comment:
Thinking about this more, it sort of defeats the purpose of the issue to not show the message about prefilling when someone isn't signed in. But we can't check the prefills available list in the profile without being signed in.

So, I guess we need to create a list of forms with prefill available in the front end code? Or add something to the form config to conditionally show this message. Right now it would just show for pension, burials, and hca.

Sorry about going back and forth on this.

Reply:
Great point, thanks for revisiting this. Changes made for option 2, which seems more consistent with how we've handled similar logic on the SIP cards. Not sure what the implications are for the existing `prefillsAvailable` logic?

Reply:
I think it can stay the way it is. I don't want to force us to make a FE change to turn prefill on for a form if we don't have to. This way, just changing the message requires a front end change.